config.py

```diff
@@ -48,7 +48,6 @@
 if github_action_db_url:
     SQLALCHEMY_DATABASE_URI = github_action_db_url
 
-SQLALCHEMY_TRACK_MODIFICATIONS = True
```

Comment:
What was the reason behind removing this config?

Reply:
I was getting error related to SQLALCHEMY_TRACK_MODIFICATIONS which got fixed after removing this.